### PR TITLE
8652: Header is named Name by default, although Title field is empty

### PR DESCRIPTION
**Pull request by savvinsergey**

### CODE REVIEW

File: src/chicletSlicer.ts [hunk 1]
```diff
@@ -670,6 +670,7 @@ module powerbi.extensibility.visual {
 
             this.slicerData = data;
             this.settings = this.slicerData.slicerSettings;
+            this.settings.header.title = this.settings.header.title.trim() || this.slicerData.categorySourceName;
```
Comment (ignatvilesov) [inline src/chicletSlicer.ts]: Can ```this.settings.header.title``` be empty?

File: test/visualTest.ts [hunk 2]
```diff
@@ -743,6 +743,13 @@ module powerbi.extensibility.visual.test {
                     expect(visualBuilder.slicerHeaderText.text()).toBe(title);
                 });
 
+                it("title default", () => {
```
Comment (ignatvilesov) [inline test/visualTest.ts]: What exactly does this test test?

File: src/chicletSlicer.ts (hunk 1 above)
Comment (savvinsergey) [inline src/chicletSlicer.ts]: No

File: test/visualTest.ts (hunk 2 above)
Comment (savvinsergey) [inline test/visualTest.ts]: This test checks default value of title if title wasn't specified

**Pull request merged**